lib/fmrest/spyke/relation.rb

```diff
@@ -268,6 +270,35 @@ def or(*params)
         params.empty? ? clone : clone.query(*params)
       end
 
+      def and(*params)
```

Comment:
There's two things I think are missing here:

1. Just like `.or` can be used without parameters to do things like `.or.query(...)` or `.or.match(...)`, I think `#and` should have the same ability. I'd like to be able to do `.and.match(...)`
2. This method needs some YARD-style inline documentation. I can probably take care of that, but I'd like to finalize the API first